cmip6_downscaling/workflows/cmip6_pyramids.py

```diff
@@ -0,0 +1,169 @@
+import os
+
+import dask
+import datatree as dt
+import intake
+from adlfs import AzureBlobFileSystem
+from carbonplan_data.metadata import get_cf_global_attrs
+from carbonplan_data.utils import set_zarr_encoding
+from intake_esm.merge_util import AggregationError
+from ndpyramid import pyramid_regrid
+from prefect import Flow, task
+from prefect.run_configs import KubernetesRun
+from prefect.storage import Azure
+
+# import datetime
+# import prefect
+# import xarray as xr
+# from prefect.executors import DaskExecutor
+# from dask_kubernetes import KubeCluster, make_pod_spec
+
+LEVELS = 2
+TIME_SLICE = slice('1950', '2100')
+# TIME_SLICE = slice('2010', '2020')
+PIXELS_PER_TILE = 128
+
+
+image = "carbonplan/cmip6-downscaling-prefect:2021.12.06"
```

Comment:
A bunch of config settings now live in cmip6_downscaling/config/config.py if you want to use those.